server/testutils/factories/person.ts

```diff
@@ -9,6 +9,9 @@ export default Factory.define<Person>(() => ({
   name: faker.name.fullName(),
   dateOfBirth: DateFormats.formatApiDate(faker.date.past()),
```

Comment:
This is from another PR, but just to point out that to be usefully realistic perhaps we should ensure that `dateOfBirth` is at least 18 years in the past?

On a general point I wonder how useful it is to have these person fixtures built dynamically with randomised properties. Would it be better to have the attributes hardcoded? And then to develop new specialised fixtures for different people as we need them eg:

- man in community
- trans woman in prison

Reply:
Yeah, we've not really thought too much about realistic fixture data, as the fixtures as a means rather than an end, but this might be a good idea for another PR 👍